test/rebar_alias_SUITE.erl

```diff
@@ -31,6 +31,7 @@ command(Config) ->
                                    ["test"], {ok, [{app, Name}]}),
     %% unlock job also ran
     Lockfile = filename:join(?config(apps, Config), "rebar.lock"),
+    io:format(user, "~p ~p Lockfile: '~p' ~n", [?MODULE, ?LINE, Lockfile]),
```

Comment:
drop this one line though, or make it a `ct:pal` so that we can avoid seeing it on every run.

Reply:
Sorry I forgot to remove it.  I've deleted it now and squashed.